### PR TITLE
#142 Made unified deployment

### LINKED ISSUE #142: Make a single installation command
The single CLI command should install both the language container and the scripts.
It should not ask for the input of the `language-alias` as it's a constant in this project.

### PR DESCRIPTION
**Pull request by ahsimb**
closes #142

### CODE REVIEW

File: exasol_sagemaker_extension/deploy.py [hunk 1]
```diff
@@ -1,28 +1,55 @@
...
+ver_formatter = ParameterFormatters()
+ver_formatter.set_formatter(CONTAINER_URL_ARG, SmeLanguageContainerDeployer.SLC_URL_FORMATTER)
+ver_formatter.set_formatter(CONTAINER_NAME_ARG, SmeLanguageContainerDeployer.SLC_NAME)
+formatters = {StdParams.version: ver_formatter}
 
-slc_parameter_formatters.set_formatter(CustomizableParameters.container_url,
-                                       SmeLanguageContainerDeployer.SLC_URL_FORMATTER)
-slc_parameter_formatters.set_formatter(CustomizableParameters.container_name,
-                                       SmeLanguageContainerDeployer.SLC_NAME)
+opts = select_std_options([StdTags.DB, StdTags.BFS, StdTags.SLC],
```
Comment (tkilias) [inline exasol_sagemaker_extension/deploy.py:21]: I guess the alias option is part of SLC standard options

File: exasol_sagemaker_extension/deploy.py [hunk 2]
```diff
@@ -1,28 +1,55 @@
...
+    if deploy_slc:
+        # Workaround for the issue#78 in PEC
+        if StdParams.path_in_bucket.name in kwargs and kwargs[StdParams.path_in_bucket.name] is None:
+            kwargs[StdParams.path_in_bucket.name] = ''
+
+        slc_deployer = LanguageContainerDeployerCli(
+            container_url_arg=CONTAINER_URL_ARG,
+            container_name_arg=CONTAINER_NAME_ARG)
+
+        extra_params = {StdParams.language_alias.name: 'PYTHON3_SME'}
```
Comment (tkilias) [inline exasol_sagemaker_extension/deploy.py:40]: Ah, ok you hard code the alias for the moment. Until we fixed the scripts that should be good enough.

File: exasol_sagemaker_extension/deployment/language_container.py [hunk 3]
```diff
@@ -16,13 +16,16 @@ def language_container_factory():
...
     with language_container_factory() as container_builder:
-        container_builder.export(export_dir)
+        export_result = container_builder.export(export_dir)
+        export_info = export_result.export_infos[str(container_builder.flavor_path)]["release"]
+        yield export_info.cache_file
 
 
 if __name__ == '__main__':
-    export_slc()
+    with export_slc(sys.argv[1]):
```
Comment (tkilias) [inline exasol_sagemaker_extension/deployment/language_container.py:30]: Ah, as a quick workaround, because we don't have nox yet setup

**Review by tkilias — DISMISSED** [review]

File: exasol_sagemaker_extension/deployment/language_container.py (hunk 3 above)
Comment (ahsimb) [reply exasol_sagemaker_extension/deployment/language_container.py:30]: Yes, basically it was like this before. I just changed the `export_slc` function slightly for the convenience of testing.

File: exasol_sagemaker_extension/deploy.py (hunk 2 above)
Comment (ahsimb) [reply exasol_sagemaker_extension/deploy.py:40]: Yes, exactly

**Review by tkilias — APPROVED** [review]

**Pull request merged**